src/pages/Integrations/Create/schema.ts

```diff
@@ -15,33 +15,96 @@ export const schema = (category) => ({
         'Configure integrations between third-party tools and the Red Hat Hybrid Cloud Console.',
       name: 'add-integration-wizard',
       fields: [
-        integrationsStep(category),
+        ...(['Communications'].includes(category)
+          ? [integrationsStep(category)]
+          : []),
         {
           title: 'Enter details',
           name: 'details',
           nextStep: 'review',
           fields: [
-            {
-              component: componentTypes.TEXT_FIELD,
-              name: 'integration-name',
-              type: 'text',
-              label: 'Integration name',
-              isRequired: true,
-              validate: [
-                {
-                  type: validatorTypes.REQUIRED,
-                },
-              ],
-            },
+            ...(['Webhooks'].includes(category)
```

Comment:
can you extract this step to its own file?

Reply:
I did this on purpose because some of these steps may be shared with your or Alex's category. So anyone following along can recycle any of these steps and put in some sort of shared array. I didn't have time to go through all the fields in all the categories and pick the shared ones since there is a PH tomorrow and the deadline is coming.

Reply:
Yeah, I can use all of this with a few conditions added.